transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollSocketChannelConfig.java

```diff
@@ -550,29 +551,21 @@ public boolean isTcpQuickAck() {
     }
 
     /**
-     * Set the {@code TCP_FASTOPEN_CONNECT} option on the socket. Requires Linux kernel 4.11 or later.
-     * See
-     * <a href="https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=19f6d3f3">this commit</a>
-     * for more details.
+     * Enables client TCP fast open. {@code TCP_FASTOPEN_CONNECT} normally
+     * requires Linux kernel 4.11 or later, so instead we use the traditional fast open
+     * client socket mechanics. See this
+     * <a href="https://lwn.net/Articles/508865/">LWN article</a> for more info.
```

Comment:
and I guess, it only works from linux 3.6 onwards. its not a massive improvement in coverage... but its maybe worth it. still curious if the same tests would fail if we added the original 4.1 TCP_FASTOPEN_CONNECT to the permutations, pretty sure most of them would.